extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -474,6 +474,38 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
       return null
     }
 
+  /**
+   * Whether to generate ids for the geojson features. When enabled, the `feature.id` property will be auto
+   * assigned based on its index in the `features` array, over-writing any previous values.
+   */
+  val generateId: Boolean?
+    /**
+     * Get the GenerateId property
+     *
+     * @return Boolean
+     */
+    get() = getPropertyValue("generateId")
+
+  /**
+   * Whether to generate ids for the geojson features. When enabled, the `feature.id` property will be auto
+   * assigned based on its index in the `features` array, over-writing any previous values.
+   */
+  val generateIdAsExpression: Expression?
```

Comment:
Hmm are we sure we want to allow generate-id to be an expression? I don't see this in the style spec.

Reply:
Could you confirm? cc @alexshalamov

Reply:
generateId is just a boolean, https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#geojson-generateId

Reply:
@alexshalamov thanks! 
nice catch @nishant-karajgikar! I will remove this expression property.

Reply:
Removed in https://github.com/mapbox/mapbox-maps-android/pull/568